### ISSUE TITLE
Available space size is negative in Android 4.0.4

### ISSUE THREAD

**Issue by ericksli** [lingochamp/FileDownloader#87]
In Sony ST26i (Android 4.0.4), when downloading file, it throws the following exception to the download listener saying that the available space size is a negative number and then the download fails. It should have enough space to store the file on the device.

````
java.io.IOException: The file is too large to store, the downloaded size:  0, requirements: 993069, but the available space size: -2143387648
````

However, in Galaxy Nexus (Android 4.0.4), it does not throw this exception and it can download the file successfully.

**Comment by Jacksgong** [comment]
Sorry about that, I already have fixed this serious bug in [v0.2.2](https://github.com/lingochamp/FileDownloader/releases/tag/v0.2.2)

So, Please import `compile 'com.liulishuo.filedownloader:library:0.2.2' `

**Comment by Jacksgong** [comment]
#84
